coral/src/app/features/topics/BrowseTopics.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { screen } from "@testing-library/react";
+import { server } from "src/services/api-mocks/server";
+import { renderWithQueryClient } from "src/services/test-utils";
+import { mockTopicGetRequest } from "src/domain/topics/topics-api.msw";
+import { BrowseTopics } from "src/app/features/topics/BrowseTopics";
+import { waitForElementToBeRemoved } from "@testing-library/react/pure";
+
+describe("TopicList.tsx", () => {
```

Comment:
I have a habit to test components in a very deep integrated manner. If I would have written tests for this component, I would  have tested that the desired data is visible in those cards. You are probably a better expert on the accessibility, so could we somehow assert that we render the `Topic list` instead of just any list. Could we use some [aria-description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description) in the `<ul>` or similar.

Reply:
I like to follow the the statement: _Write tests. Not too many. Mostly integration._

However, I like the amount of tests we have in the project and there will be overlap. I would not think too much, but we could do a quick checkup after some time about the scopes once we have more functionality tested. This comment is not a hard requirement by any means, so feel free to leave the implementation as is.